scripts/load_testing/create_account.py

```diff
@@ -12,81 +13,118 @@
 def authenticity_token(dom):
     return dom.find('input[name="authenticity_token"]')[0].attrib['value']
 
-class UserBehavior(locust.TaskSet):
-    @locust.task
-    def signup(self):
-        # visit home page
-        self.client.get('/', auth=auth)
 
-        # visit create account page and submit email
-        resp = self.client.get('/sign_up/enter_email', auth=auth)
-        resp.raise_for_status()
-        dom = pyquery.PyQuery(resp.content)
-        data = {
-            'user[email]': 'test+' + fake.md5() + '@test.com',
-            'authenticity_token': authenticity_token(dom),
-            'commit': 'Submit',
-        }
-        resp = self.client.post('/sign_up/enter_email', data=data, auth=auth)
-        resp.raise_for_status()
+def signup(t):
+    # visit home page
+    t.client.get('/sign_up/start', auth=auth)
+
+    # visit create account page and submit email
+    resp = t.client.get('/sign_up/enter_email', auth=auth)
+    resp.raise_for_status()
+    dom = pyquery.PyQuery(resp.content)
+    data = {
+        'user[email]': 'test+' + fake.md5() + '@test.com',
+        'authenticity_token': authenticity_token(dom),
+        'commit': 'Submit',
+    }
+    resp = t.client.post('/sign_up/enter_email', data=data, auth=auth)
+    resp.raise_for_status()
 
-        # capture email confirmation link on resulting page
-        dom = pyquery.PyQuery(resp.content)
+    dom = pyquery.PyQuery(resp.content)
+    try:
         link = dom.find("a[href*='confirmation_token']")[0].attrib['href']
+    except IndexError:
+        print("""
+            Failed to get confirmation token. 
+            Consult https://github.com/18F/identity-idp#load-testing
+            and check your application config."""
+        )
+        return
```

Comment:
Nice, that's super helpful. What do you think about extending this further and 1.) returning the status code (I'm guessing that a lot of these errors are 503s under heavy load) and 2.) tallying the number of these errors so we know how many times it failed if the run finishes or exits early.